scripts/scil_bundle_volume.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import argparse
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_json_args,
+                             add_reference_arg,
+                             assert_inputs_exist)
+from scilpy.tractanalysis import compute_tract_counts_map
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Compute bundle volume in mm³',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file.')
+
+    add_reference_arg(p)
+    add_json_args(p)
+
+    return p
+
+
+def main():
+    parser = _build_arg_parser()
+    args = parser.parse_args()
+
+    assert_inputs_exist(parser, [args.in_bundle, args.reference])
+
+    sft = load_tractogram_with_reference(parser, args, args.in_bundle)
+    sft.to_vox()
+    sft.to_corner()
+
+    bundle_name, _ = os.path.splitext(os.path.basename(args.in_bundle))
+    stats = {bundle_name: {}}
+    if len(sft.streamlines) == 0:
+        stats[bundle_name]['volume'] = None
+        print(json.dumps(stats, indent=args.indent, sort_keys=args.sort_keys))
+        return
+
+    ref_img = nib.load(args.reference)
```

Comment:
This will crash when ref_img is not loaded, which will be the case for most usage (with trk).

This informations is in the sft.space_attribute (or as of Dipy 1.1, sft.space_attributes or sft.voxel_sizes)